src/ocean_component.cpp

```diff
@@ -299,6 +299,11 @@ void OceanComponent::prepareToRun() {
     surfaceLL.mychemistry.As            = ocean_area * part_low; // surface area m2
     surfaceLL.mychemistry.U             = 6.7; // average wind speed m/s Hartin et al. 2016
 
+    // Initialize surface flux tracking variables
+    annualflux_sum.set( 0.0, U_PGC );
+    annualflux_sumHL.set( 0.0, U_PGC );
+    annualflux_sumLL.set( 0.0, U_PGC );
```

Comment:
The annualflux_sum is also set in the run component around line 360 does it really need to be set twice? Or could we remove that code block?

Reply:
It needs to be zeroed her as well, so that the initial output visitor pass — which now occurs _before the component has ever run_ — can correctly report model state.